.github/workflows/_test_pax_rosetta.yaml

```diff
@@ -471,17 +476,17 @@ jobs:
     if: ( always() )
     secrets: inherit
     with:
-      ENDPOINT_FILENAME: 'pax-test-status.json'
+      ENDPOINT_FILENAME: '${{ inputs.ARTIFACT_NAME }}pax-test-status.json'
       PUBLISH: false
       SCRIPT: |
-        EXIT_STATUSES="${GITHUB_RUN_ID}-*DP*FSDP*TP*PP*/*-status.json ${GITHUB_RUN_ID}-*DP_TE_dropout/*-status.json"
+        EXIT_STATUSES="${{ inputs.ARTIFACT_NAME }}${GITHUB_RUN_ID}-*DP*FSDP*TP*PP*/*-status.json ${{ inputs.ARTIFACT_NAME }}${GITHUB_RUN_ID}-*DP_TE_dropout/*-status.json"
         PASSED_TESTS=$(jq -r '. | select ((.state == "COMPLETED") and (.exitcode == "0")) | .state' $EXIT_STATUSES | wc -l)
         FAILED_TESTS=$(jq -r '. | select ((.state != "COMPLETED") or (.exitcode != "0")) | .state' $EXIT_STATUSES | wc -l)
         TOTAL_TESTS=$(ls $EXIT_STATUSES | wc -l)
 
-        echo '## PAX MGMN Test Status' >> $GITHUB_STEP_SUMMARY
+        echo '## Rosetta PAX MGMN Test Status' >> $GITHUB_STEP_SUMMARY
         for i in $EXIT_STATUSES; do
-          echo $i | cut -d'.' -f1
+          echo $i | rev | cut -d'-' -f1 | rev
```

Comment:
What do you think about:
`basename -- $i -status.json`

Reply:
What does this line do?

Reply:
We simply want to extract the test name from the exit status json files. I implemented your suggestion @terrykong